dts/arm/nxp/nxp_rt118x_cm33_ns.dtsi

```diff
@@ -22,8 +22,8 @@
 			zephyr,memory-region = "DTCM";
 		};
 
-		memory: memory@20000000 {
-			ranges = <0x0 0x20000000 0x10000000>;
+		memory: memory@20484000 {
+			ranges = <0x484000 0x20484000 0x10000000>;
```

Comment:
How did you pick these values. OCRAM from the Reference Manual is from `0x20480000` and not `0x20484000`.

Reply:
Also you are changing the start point from `0x0` to `0x484000`. Please update the `reg` field for the `ocram` nodes inside `nxp_rt118x.dtsi`

Reply:
> How did you pick these values. OCRAM from the Reference Manual is from `0x20480000` and not `0x20484000`.
Yes, however first 16K access is blocked by TRDC, this is aligned with SDK. You can see the comment in nxp_rt118x.dtst 937 line like below: 
/* OCRAM1 first 16K access is blocked by TRDC */

Reply:
> Also you are changing the start point from `0x0` to `0x484000`. Please update the `reg` field for the `ocram` nodes inside `nxp_rt118x.dtsi`

Thank you for pointing out, I have updated the reg address.